tests/unittest_filter_statements.py

```diff
@@ -0,0 +1,20 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
+
+import unittest
```

Comment:
I'd prefer moving `astroid` to `pytest` as well so I tend to change test to `pytest` when I encounter them. However, not sure what the "official policy" is?

Reply:
Makes sense if `pytest` is being used for test discovery anyway. unittest test discovery currently doesn't work.

Reply:
Perhaps. I do kind of worry about the contribution barrier if we no longer accept tests written in unittest. There are a number of things I know how to do in unittest that I don't in pytest because I don't work with it ever.